### ISSUE TITLE
Can't build weblaf 2.9 - WebLanguageManager must be initialized first

### ISSUE THREAD

**Issue by samuelvisser** [mgarin/weblaf#496]
I recently switched over to JFormDesigner as my UI editor. Had to use the weblaf 1.2.9 prerelease because it couldn't find classes (see [this issue](https://github.com/mgarin/weblaf/issues/123)). That works now, but I'm getting this new error when trying to build and run the project:

```
com.alee.managers.language.LanguageException: WebLanguageManager must be initialized first
	at com.alee.managers.language.UILanguageManager.mustBeInitialized(UILanguageManager.java:265)
	at com.alee.managers.language.UILanguageManager.getInitialText(UILanguageManager.java:559)
	at com.alee.laf.button.WebButton.init(WebButton.java:295)
	at javax.swing.JButton.<init>(JButton.java:137)
	at com.alee.laf.button.WebButton.<init>(WebButton.java:284)
	at com.alee.laf.button.WebButton.<init>(WebButton.java:177)
	at com.alee.extended.statusbar.WebMemoryBar.<init>(WebMemoryBar.java:81)
	at com.alee.extended.statusbar.WebMemoryBar.<init>(WebMemoryBar.java:76)
	at com.openbravo.pos.forms.JRootApp.initComponents(JRootApp.java:984)
	at com.openbravo.pos.forms.JRootApp.<init>(JRootApp.java:206)
	at com.openbravo.pos.forms.JRootFrame.initFrame(JRootFrame.java:70)
	at com.openbravo.pos.forms.StartPOS$1.run(StartPOS.java:132)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

Using Intelij IDEA with JFormDesigner plugin.

**Comment by mgarin** [comment]
I suppose the later versions (as you commented on #123) fix that issue for you?

**Comment by samuelvisser** [comment]
I might have found a different solution. Do you know what files are hosted at [this mvn repository](https://mvnrepository.com/artifact/de.sciss/weblaf)? The 1.2.9 version there seems to fix the building problems. For some reason, 1.2.9 seems to be mutch the same as your 1.28, exept for all the problems I was having :). The problem from [issue 123](https://github.com/mgarin/weblaf/issues/123) returned, but this time only in the JFormDesigner UI, the build worked. I fixed the JFormDesigner UI by replacing the compiled XmlUtils.class in the jar file from 1.2.9 with the same file from 1.28 but with the issue 123 fix implemented. 

It works now, but would like to know if you know this repository and where I can find the source code for it. I only have the compiled jar for now

**Comment by mgarin** [comment]
This should not be happening anymore with recently released v1.2.9 update, so i'll be closing this issue.
Feel free to reopen if it still occurs for you.
